docs/references/architecture/adr-113-txi.md

```diff
@@ -0,0 +1,97 @@
+# ADR 113: Modular transaction hashing
+
+## Changelog
+
+- 2024-02-05: First version (@melekes)
+
+## Status
+
+TBD
+
+## Context
+
+Transaction hashing in CometBFT is currently implemented using `crypto/tmhash`
```

Comment:
Is this ADR meant only for transactions or also for the `Block` hash?

Reply:
At the moment, only for transaction's hash. I would make it very precise in the description.

Reply:
This is up for discussion. @fedekunze would you benefit from changing `Block` hash?